src/SDKs/ServiceBus/data-plane/Directory.Build.props

```diff
@@ -28,4 +22,14 @@
     -->
     <DefineConstants Condition="'$(OS)' == 'Windows_NT'">$(DefineConstants);WINDOWS</DefineConstants>
   </PropertyGroup>
+
+  <!-- 
+    For Track 1, override the engineering system signing key and use the existing key for
+    Azure messaging client libraries.
+  -->
+  <PropertyGroup Condition="'$(SignAssembly)' != 'false'">
```

Comment:
Do we need another key? Why not use `eng/MSSharedLibKey`?

Reply:
In short:  to preserve binary compatibility with the existing packages for Track 1 libraries.   This decision was made in collaboration with our engineering team and the service team.

Reply:
My bad.

Reply:
All good.   It's a reasonable question to ask.   :)